Dockerfile.template

```diff
@@ -240,10 +245,14 @@ RUN set -eux; \
 		| sort -u \
 		| xargs -r apt-mark manual \
 	; \
+# add cloud-init support 
+        apt-mark manual genisoimage; \
 	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
 	\
 # basic smoke test
-	qemu-img --version
+	qemu-img --version; 
+COPY --from=cloudutilssrc /usr/bin/cloud-localds /usr/bin/cloud-localds 
```

Comment:
maybe add this comment:
#copy cloud-init support, form cloudutilssrc, to use it together with genisoimage